src/repl/repl.ts

```diff
@@ -1,24 +1,30 @@
-import repl = require('repl')
```

Comment:
Try not to make refactors in the same PR as a feature PR. It adds additional noise to the PR which reviewers would have to filter out.

Reply:
Noted, I'll keep that in mind for future PRs.